dvc/tree/azure.py

```diff
@@ -92,6 +93,8 @@ def blob_service(self):
             container_client.get_container_properties()
         except ResourceNotFoundError:
             container_client.create_container()
+        except HttpResponseError:
```

Comment:
Maybe we could catch a more specific exception here? Or are you getting a generic HttpResponseError? If so, what code does have?

Reply:
It's a generic one unfortunately, status code is 403:

```
(Pdb) e
HttpResponseError('This request is not authorized to perform this operation.\nRequestId:xyz\nTime:2020-09-21T10:16:42.1106259Z\nErrorCode:AuthorizationFailure\nError:None')
(Pdb) dir(e)
['__cause__', '__class__', '__context__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__setstate__', '__sizeof__', '__str__', '__subclasshook__', '__suppress_context__', '__traceback__', '__weakref__', '_parse_odata_body', 'additional_info', 'args', 'error', 'error_code', 'exc_msg', 'exc_traceback', 'exc_type', 'exc_value', 'inner_exception', 'message', 'model', 'raise_with_traceback', 'reason', 'response', 'status_code', 'with_traceback']
(Pdb) e.status_code
403
(Pdb) e.reason
'This request is not authorized to perform this operation.'
```

Reply:
@mhworth That's actually pretty good, `403` is `Forbidden`, better than nothing. Let's check that this is indeed 403. Otherwise looks good.

Reply:
Sounds good! Just pushed up an edit that checks to see if status_code == 403 and then reraises if not.